include/rcutils/env.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Open Source Robotics Foundation, Inc.
+// Copyright 2017-2021 Open Source Robotics Foundation, Inc.
```

Comment:
Let's not change this; we haven't been updating it anywhere else.  Same goes for all of the copyright dates below.

Reply:
Alright, I reverted 4eb6f857ba63e3e26b8f2384c10031cff7f3a399: fed8ca8fdf7094adb99ce2e936162c64f78dfffa